libraries/package-deps-hash/src/getRepoState.ts

```diff
@@ -282,7 +282,13 @@ async function spawnGitAsync(
   });
 
   if (stdin) {
-    stdin.pipe(proc.stdin!);
+    /**
+     * We are just interested in catching "error" events there from child process'
+     * proc.stdin to avoid stdin based git commands to throw unhandled exceptions
+     * hence to let a chance for the error to be thrown in a context we can recover
+     * from.
```

Comment:
```suggestion
     * For `git hash-object` data is piped in asynchronously. In the event that one of the
     * passed filenames cannot be hashed, `proc.stdin` will be closed, causing writes to fail.
     * Silence this error since it will be handled by the non-zero exit code of the process.
```